src/main/java/com/management/model/CarModel.java

```diff
@@ -6,26 +6,28 @@
 import jakarta.persistence.Id;
 
 @Entity
-public class Car {
+public class CarModel {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String brand;
     private String model;
+    private String plate;
     private Integer year;
 
     // Constructors, getters, and setters
-    public Car() {
+    public CarModel() {
     }
 
-    public Car(String brand, String model, Integer year) {
+    public CarModel(String brand, String model, String plate, Integer year) {
         this.brand = brand;
         this.model = model;
         this.year = year;
+        this.plate = plate;
```

Comment:
Só uma sugestão para melhorar o entendimento aqui.

```suggestion
        this.license_plate = license_plate ;
```

Reply:
Done, the changes were made

        modified:   src/main/java/com/management/controller/CarController.java
        modified:   src/main/java/com/management/model/CarModel.java
        modified:   src/main/java/com/management/repository/CarRepository.java
        modified:   src/main/java/com/management/service/CarService.java
        
        to support a better name to licensePlate